openc3/lib/openc3/logs/log_writer.rb

```diff
@@ -40,6 +40,11 @@ class LogWriter
     #   independently.
     attr_reader :cycle_time
 
+    # @return cycle_size [Integer] The amount of data in bytes before creating
+    #   a new log file. This can be combined with cycle_time but is better used
+    #   independently.
```

Comment:
I don't think it is better used independenly.  The ideal is either, time or size, whichever comes first.

Reply:
Copy / paste. I'll change the other comments.